packages/examples/packages/interactive-ui/src/index.ts

```diff
@@ -0,0 +1,331 @@
+import { rpcErrors } from '@metamask/rpc-errors';
+import type {
+  OnHomePageHandler,
+  OnTransactionHandler,
+  OnUserInputHandler,
+  Transaction,
+  Component,
+} from '@metamask/snaps-sdk';
+import {
+  panel,
+  type OnRpcRequestHandler,
+  heading,
+  button,
+  UserInputEventType,
+  form,
+  input,
+  ButtonType,
+  text,
+  copyable,
+  ManageStateOperation,
+  assert,
+  address,
+  row,
+} from '@metamask/snaps-sdk';
+import { hasProperty } from '@metamask/utils';
+
+import { decodeData } from './utils';
+
+/**
+ * Initiate a new interface with the starting screen.
+ *
+ * @returns The Snap interface ID.
+ */
+export async function createInterface(): Promise<string> {
```

Comment:
Maybe it would be a little bit simpler and easier to read if we move some of these to a `ui` file. WDYT?